### DISCUSSION TITLE
Installation on arch?

### DISCUSSION THREAD

**Discussion by cubuzz** [udevbe/greenfield#61] [Q&A]
Hi! I'm trying to install greenfield on arch. Can someone provide a full step-by-step guide? And if so, feel free to put it on the archwiki as well!

Thanks!

**Comment by Zubnix** [comment]
Hi cubuzz, I'm no arch user but I might help/guide you.

Are there specifics you are struggling with or do you want perhaps a step-by-step guide starting from zero?

**Reply by cubuzz** [reply]
Hi! There are indeed only some specifics I'm struggling with. Likely it's something simple I'm missing :,) Those issues arise with the streaming library thingy not being found, despite explicit installation. I'll try to recreate the issue on Saturday, will keep ya posted!

**Reply by cubuzz** [reply]
The dockerfile mentioned in #62 helped me fix my stupidity. Once again, thank you for your support.

**Comment by 3ri4nG0ld** [comment]
Is there any news on this issue?
I am unable to install greenfield manually.
